app/models/ahoy_captain/i18n_config.rb

```diff
@@ -0,0 +1,23 @@
+module AhoyCaptain
+  class I18nConfig < ::I18n::Config
+    BACKEND = I18n::Backend::Simple.new
+    AVAILABLE_LOCALES = AhoyCaptain::Engine.root.join("config/locales/ahoy_captain").glob("*.yml").map { |path| File.basename(path, ".yml").to_sym }.uniq
+    AVAILABLE_LOCALES_SET = AVAILABLE_LOCALES.inject(Set.new) { |set, locale| set << locale.to_s << locale.to_sym }
+
+    def backend
+      BACKEND
+    end
+
+    def available_locales
```

Comment:
Why do we have two methods, `available_locales` and `available_locales_set`? Why not consolidate into a single `available_locales` method?